integration-tests/src/main/resources/META-INF/services/tachyon.underfs.UnderFileSystemFactory

```diff
@@ -0,0 +1,2 @@
+tachyon.underfs.hdfs.HdfsUnderFileSystemFactory
+tachyon.underfs.glusterfs.GlusterFSUnderFileSystemFactory
```

Comment:
Removed. It turns out by including tachyon-underfs-hdfs I don't really need this file.

Thanks for the catch.